capture-sdk/sdk/src/main/java/net/gini/android/capture/ClickListenerExt.kt

```diff
@@ -0,0 +1,7 @@
+package net.gini.android.capture
+
+import android.view.View
+
+fun View.onClick(click: (View) -> Unit) {
```

Comment:
[Request] Let's make it clear what this method does by renaming it to `View.setIntervalClickListener`.